### ISSUE TITLE
Updates to the DMP-ID landing page

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmphub#124]
This is an epic to track work updating & refining the DMP-ID landing page

**Comment by briri** [comment]
Option A, current version: https://wzfpyk.axshare.com/#id=6nw33p&p=dmp-id-landing-page_droplist_header_a&c=1
Option A, older version: https://wzfpyk.axshare.com/#id=13iheq&p=dmp-id-landing-page_droplist_header_a_old-version&c=1

As far as colors go, I think the best fix for accessibility is to make the body text pure black. In that case, the current blue and green are both fine– but DMPTool uses a slightly lighter blue, so you might want to update to match. That scheme would be:
Black: #000000
Blue: #105e8e
Green: #0e8256

**Comment by briri** [comment]
closing since this is already deployed to DMPHub production
